doc/tools.md

```diff
@@ -112,3 +114,14 @@ Removes the zip files in the <tt>tmp</tt> folder that's created when you package
 
     $ zat clean
 
+### Create
+
+Packages your app directory into a zipfile, then uploads it into a Zendesk account. Also stores the new app's ID and other metadata in a zatfile (by default, this is `.zat`).
+
+A `path` option pointing to the directory containing the app can be specified on the command-line using `zat create --path=./MY_PATH`. The name of the `zatfile` can be passed using `zat create --zatfile=.zat.test`.
```

Comment:
Original:
> A `path` option pointing to the directory containing the app can be specified on the command-line using `zat create --path=./MY_PATH`. The name of the `zatfile` can be passed using `zat create --zatfile=.zat.test`.

Sentence structured as "can be this" and "can be that", which is okay but not optimal in terms of the tasks the dev is trying to accomplish ("I want to this" and "I want to do that"). Suggested edit to make it a little more specific:

> You can point to the directory containing the app by using the `path` option. Example: `zat create --path=./MY_PATH`. You can specify the name of the zat file with the `zatfile` option. Example: `zat create --zatfile=.zat.test`.

* command-line -> command line (no hyphen if used as a noun)

* "the name of the `zatfile" -> "the name of the zat file" -> 2 words and no code-case if referring to a file, not the option